adafruit_bmp280.py

```diff
@@ -502,15 +515,17 @@ class Adafruit_BMP280_SPI(Adafruit_BMP280):
 
     """
 
-    def __init__(self, spi, cs, baudrate=100000):
+    def __init__(
+        self, spi: SPI, cs: Optional[DigitalInOut] = None, baudrate=100000
```

Comment:
`SPIDevice` _does_ allow for `None` as a chip select, but I don't think the sensor will function properly if not given... it may be better to just type it as `DigitalInOut`.  @FoamyGuy, what do you think?

Reply:
I think that is correct, I would assume it will not communicate properly without a CS pin. The SPI code in the examples (commented out by default) does show passing in a DigitalInOut in each of the places that I found.

I would agree with making it `DigitalInOut` type and not having the default value of `None`. 

Thanks for working on these type annotations @billvanleeuwen424!